src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -74,7 +74,8 @@ CHIP_ERROR CASEServer::InitCASEHandshake(Messaging::ExchangeContext * ec)
     ReturnErrorOnFailure(mIDAllocator->Allocate(mSessionKeyId));
 
     // Setup CASE state machine using the credentials for the current fabric.
-    ReturnErrorOnFailure(GetSession().ListenForSessionEstablishment(mSessionKeyId, mFabrics, this));
+    ReturnErrorOnFailure(GetSession().ListenForSessionEstablishment(
+        mSessionKeyId, mFabrics, this, Optional<ReliableMessageProtocolConfig>::Value(gDefaultMRPConfig)));
```

Comment:
This argument list is growing and growing. Could it be replaced by a struct?

Reply:
Maybe in a separate PR?